calcom/cal.com
# fix: add metadata to stripe payment intent

## What does this PR do?

- Add booking and event metadata to payment intent stripe

Fixes #10678 

## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?



- Are there environment variables that should be set? No
- What are the minimal test data to have? Stripe app setup on event type
- What is expected (happy path) to have (input and output)? In stripe dashboard payment intent you should see new metadata to identify that this payment came from cal.com.
- Any other important info that could help to test that PR. No

## Mandatory Tasks


## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
